src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -281,6 +281,7 @@ public static class Dungeons {
         public int mapX = 2;
         public int mapY = 2;
         public boolean solveThreeWeirdos = true;
+        public boolean repartyRejoin = true;
```

Comment:
This option should be placed elsewhere since the order of the fields determine where the option is placed in the config and this spot doesn't make much sense

Reply:
Fair enough, I'm considering putting it in "General" options instead of "Dungeons" because it is commonly used feature not only for Dungeons but also for Kuudra